docs/guide/data_types/timeseries.rst

```diff
@@ -1,196 +1,95 @@
 
-sunpy will attempt to detect automatically the instrument source for most FITS
-files. However timeseries data are stored in a variety of file types (FITS, txt,
-csv) and formats, and so it is not always possible to detect the source. For
-this reason, it is good practice to explicitly state the source for the file.
-sunpy ships with a number of known instrumental sources.  If you would like
-sunpy to include another instrumental source please follow the
-`sunpy contribution guide <https://sunpy.org/contribute>`__.
-The `~sunpy.timeseries.TimeSeries` factory has the ability to make a list of
-TimeSeries objects using a list of filepaths, a folder or a glob, for example: ::
+SunPy will attempt to detect automatically the instrument source for most FITS files.
+However timeseries data are stored in a variety of file types (FITS, txt, csv) and formats, and so it is not always possible to detect the source.
+For this reason, it is good practice to explicitly state the source for the file.
+SunPy ships with a number of known instrumental sources.
+If you would like SunPy to include another instrumental source please follow the `SunPy contribution guide <https://sunpy.org/contribute>`__.
+The `~sunpy.timeseries.TimeSeries` factory has the ability to make a list of TimeSeries objects using a list of filepaths, a folder or a glob, for example: ::
 
     >>> my_ts_list = ts.TimeSeries('filepath1', 'filepath2', source='XRS')   # doctest: +SKIP
     >>> my_ts_list = ts.TimeSeries('/goesdirectory/', source='XRS')   # doctest: +SKIP
     >>> my_ts_list = ts.TimeSeries(glob, source='XRS')   # doctest: +SKIP
 
-Note that this functionality will only work with files from the same single
-source, generating a source specific child of the `~sunpy.timeseries.GenericTimeSeries`
-class such as the `~sunpy.timeseries.sources.goes.XRSTimeSeries` above. For this
-reason, all the files should be from that same source for the `~sunpy.timeseries.TimeSeries`
-factory to work as intended.
+Note that this functionality will only work with files from the same single source, generating a source specific child of the `~sunpy.timeseries.GenericTimeSeries` class such as the `~sunpy.timeseries.sources.goes.XRSTimeSeries` above.
+For this reason, all the files should be from that same source for the `~sunpy.timeseries.TimeSeries` factory to work as intended.
 
 1.1 Creating a Single TimeSeries from Multiple Files
-====================================================
+----------------------------------------------------
 
-You can create a single time series from multiple files for a given source using
-the keyword argument ``concatenate=True``, such as:
+You can create a single time series from multiple files for a given source using the keyword argument ``concatenate=True``, such as:
 
     >>> my_timeseries = ts.TimeSeries('/mydirectory/myts1.fits', '/mydirectory/myts2.fits', source='XRS', concatenate=True)  # doctest: +SKIP
```

Comment:
```suggestion
    >>> my_timeseries = ts.TimeSeries(sunpy.data.sample.GOES_XRS_TIMESERIES, sunpy.data.sample.GOES_XRS_TIMESERIES, source='XRS', concatenate=True)  # doctest: +REMOTE_DATA
```
I wonder what this would do compared to the above example, it probably will be the same.